starter-code/index.html

```diff
@@ -1,31 +1,53 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <title>Instagram</title>
+  <link rel="stylesheet" href="css/main.css">
 </head>
+
 <body>
-  Instagram
+  <div class="background">
+  <div class="right-side">
+    <div class="header">
+      <img src="images/brand.png">
+      <button id="login-button">
+        <img src="images/home.png">
+        <p>Log in</p>
+      </button>
+    </div>
+    <!--Instagram-->
+    <div class="main">
+      <h1>Capture and Share the World's Moments</h1>
 
-  Capture and Share the World's Moments
+      <p>Instagram is a <strong>fast</strong>, <strong>beautiful</strong> and <strong>fun</strong> way to share your life with friends and family.</p>
 
-  Instagram is a fast, beautiful and fun way to share your life with friends and family.
+      <p>Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.</p>
 
-  Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
+      <p>Oh yeah, did we mention it's free?</p>
 
-  Oh yeah, did we mention it's free?
+      <button class="downloadButton"><img src="images/badge-iphone.png"></button>
+      <button class="downloadButton"><img src="images/badge-android.png"></button>
 
-  Download on the App Store
-  Get it on Google Play
+    </div>
 
-  About Us
-  Support
-  Blog
-  Press
-  API
-  Jobs
-  Privacy
-  Terms
 
-  &copy; 2014 Instagram
+  </div>
+  <div class="left-side">
+    <img src="images/phones.png">
+  </div>
+  <div class="footer">
```

Comment:
Example of how to improve it (you may have to tweak it abit to have it work in your page).
```
  <footer>
    <ul>
      <li>
        <a href="#">About Us</a>
      </li>
      <li>
        <a href="#">Support</a>
      </li>
      <li>
        <a href="#">Blog</a>
      </li>
      <li>
        <a href="#">Press</a>
      </li>
      <li>
        <a href="#">API</a>
      </li>
      <li>
        <a href="#">Jobs</a>
      </li>
      <li>
        <a href="#">Privacy</a>
      </li>
      <li>
        <a href="#">Terms</a>
      </li>
      <li>&copy; 2014 Instagram</li>
    </ul>
  </footer>
```


css footer:

```
footer ul {
  clear: both;
  list-style: none;
  margin: 0 0 20px;
  padding: 0;
  text-align: center;
}

footer ul li {
  color: #999;
  display: inline;
  font-size: 11px;
  font-weight: bold;
  margin: 0 10px;
  text-transform: uppercase;
}

footer ul li a {
  color: #1c5380;
  text-decoration: none;
}

footer ul li a:hover {
  text-decoration: underline;
}
```